### PR TITLE
chore: switch to true static attributes

### PR DESCRIPTION
**Pull request by wraithgar**
now that we're off of node10 we can clean this up

### CODE REVIEW

**Comment by isaacs** [comment]
Probably not a concern in this case, but something to be aware of when switching from `static get foo () { return 'bar' }` to `static foo = 'bar'` is that it's no longer write-protected.

File: lib/commands/access.js [hunk 1]
```diff
@@ -20,35 +20,23 @@ const subcommands = [
...
-      '2fa-required [<package>]',
-      '2fa-not-required [<package>]',
-      'ls-packages [<user>|<scope>|<scope:team>]',
-      'ls-collaborators [<package> [<user>]]',
-      'edit [<package>]',
-    ]
-  }
+  static description = 'Set access level on published packages'
+  static name = 'access'
+  static params =[
```
Comment (ljharb) [inline lib/commands/access.js]:
```suggestion
  static params = [
```

Comment (wraithgar) [reply lib/commands/access.js]: Once linting is working the linter will catch these.

**Comment by wraithgar** [comment]
We can't move to this until we move onto `@npmcli/eslint-config` which is a larger task than just switching, because there are several bugs that were fixed there that aren't fixed here, such as line length enforcement being ignored in almost 250 places here.

**Comment by wraithgar** [comment]
> Probably not a concern in this case, but something to be aware of when switching from `static get foo () { return 'bar' }` to `static foo = 'bar'` is that it's no longer write-protected.

Not in the constructor, but it's protected in the "blessed" api i.e. in the instantiated object.

https://github.com/npm/cli/blob/latest/lib/base-command.js#L12-L18

If someone requires the base class and overrides the constructor there that's ... on them?

**Comment by ljharb** [comment]
why do you need to wait for the eslint config? you should be able to set the parserOptions.ecmaVersion to 2022 in the meantime.

**Comment by wraithgar** [comment]
eslint@7 doesn't support it at all, and updating to 8 in this repo would have meant auditing the other eslint plugins and updating this local eslint config, something that had already been done in our linting repo.  We already moved the linting today so it's just a matter of landing the breaking change to `@npmcli/eslint-config`

**Comment by wraithgar** [comment]
https://github.com/npm/eslint-config/pull/10

**Comment by ljharb** [comment]
ah, true, you'd need to use `@babel/eslint-parser` with eslint 7.

**Comment by isaacs** [comment]
@wraithgar 

> Not in the constructor, but it's protected in the "blessed" api i.e. in the instantiated object.

Ah, ok, I'd missed that.  Since we don't support loading npm programmatically any more in v8, I really can't see any problem.  It's easy enough for us to just not do the wrong thing.

**Review by lukekarrys — APPROVED** [review]

**Pull request merged**